doc/introduction.rst

```diff
@@ -1,11 +1,3 @@
-.. for doc tests to run with recent NumPy 1.14, we need to set print options
```

Comment:
is this not the case anymore?

Reply:
Now we support numpy>=1.19. Plus there are no more doc tests in this file